api/generated/informers/externalversions/nfd/v1alpha1/nodefeature.go

```diff
@@ -62,7 +62,23 @@ func NewFilteredNodeFeatureInformer(client versioned.Interface, namespace string
 				if tweakListOptions != nil {
 					tweakListOptions(&options)
 				}
-				return client.NfdV1alpha1().NodeFeatures(namespace).List(context.TODO(), options)
+				if options.Limit == 0 {
```

Comment:
this works but is editing a generated file, anyone know a good clean way to have custom informer factory logic here? I could make a wrapper outside of the generated directory but not sure what the cleanest mechanism here is

Reply:
ah just tested by only setting limit without changing the file, based on kubernetes audit logs it looks like the informer handles the continue stuff for you :)